content/300-guides/300-upgrade-guides/800-upgrade-from-prisma-1/08-upgrade-from-mongodb-beta.mdx

```diff
@@ -110,7 +110,7 @@ Take caution in doing these renames because you need to make sure the Prisma Sch
 
 Unlike SQL databases, MongoDB doesn't have an explicit understanding of relationships between data. This means that Prisma's introspection is unable to infer those relationships for you.
 
-We typically recommend adding the relationships by hand with the help of [this documentation](https://prisma.io/docs/guides/database/using-prisma-with-mongodb#how-to-add-in-missing-relations-after-introspection). However, Prisma 1 stores foreign keys is different than where Prisma 2 and later expects foreign keys, so if you want to take advantage of relationships, you'll need to shift where the foreign keys are on your database before adding the relationships.
+We typically recommend adding the relationships by hand with the help of [this documentation](https://www.prisma.io/docs/guides/database/using-prisma-with-mongodb#how-to-add-in-missing-relations-after-introspection). However, Prisma 1 stores foreign keys is different than where Prisma 2 and later expects foreign keys, so if you want to take advantage of relationships, you'll need to shift where the foreign keys are on your database before adding the relationships.
```

Comment:
This should not be an absolute URL if it links to another docs page, but an internal link.

Reply:
I honestly do not know - I would need to look up how we do all the other internal links in the docs. Should be easy enough to find some, possible in the same file.

Reply:
Checked with previous files we had (i.e. `01-quickstart.mdx` and URLs relative from docs start **without** `/docs/`. Updated files with this and removed redundant url changes to other sources. Thanks for the check!